docs/requirements.md

```diff
@@ -10,13 +10,13 @@ Running Codacy on a Kubernetes cluster requires the following:
 
 The resource usage for Codacy depends a lot on the amount of concurrent analysis that you configure. Check the following table for some examples.
 
-| Installation type    | Max Parallel Analysis | vCPUs | Memory | Storage |
-| -------------------- | --------------------- | ----- | ------ | ------- |
-| MicroK8s Minimum     | 2                     | 16    | 32 GB  | 1TB+    |
-| MicroK8s Recommended | 2                     | 16    | 64 GB  | 1TB+    |
-| Small Installation   | 2                     | 16    | 64 GB  | 1TB+    |
-| Medium Installation  | 4                     | 32    | 128 GB | 1TB+    |
-| Big Installation     | 10                    | 64    | 256 GB | 1TB+    |
+| Installation type              | Max. parallel analysis | vCPUs | Memory | Storage |
+| ------------------------------ | ---------------------- | ----- | ------ | ------- |
+| Kubernetes Small Installation  | 2                      | 16    | 64 GB  | 1 TB+   |
+| Kubernetes Medium Installation | 4                      | 32    | 128 GB | 1 TB+   |
+| Kubernetes Big Installation    | 10                     | 64    | 256 GB | 1 TB+   |
+| MicroK8s Minimum               | 2                      | 16    | 32 GB  | 1 TB+   |
+| MicroK8s Recommended           | 2                      | 16    | 64 GB  | 1 TB+   |
```

Comment:
@paulopontesm I made it more explicit that some lines apply to MicroK8s while the others apply to "normal" Kubernetes clusters.

I also decided to place the MicroK8s at the bottom of the table because as far as I know this solution is not as scalable and should only be considered for particular situations where the performance will not degrade the experience of using Codacy.

Reply:
We should relate the number of parallel analysis with commits/hour?